source/api/commands/route2.md

```diff
@@ -0,0 +1,449 @@
+---
+title: route2
+containerClass: experimental
+---
+
+{% note warning %}
+{% fa fa-warning orange %} **This is an experimental feature. In order to use it, you must manually configure the {% url "`experimentalNetworkMocking`" experiments %} option to `true`.** See {% issue 687 %} for more details.
+{% endnote %}
+
+Use `cy.route2()` to manage the behavior of network requests at the network layer.
+
+# Comparison to `cy.route()`
+
+Unlike {% url `cy.route()` route %}, `cy.route2()`:
+
+- can mock all types of network requests including Fetch API, page loads, XMLHttpRequests, etc.
+- does not require calling {% url `cy.server()` server %} before use
+
+# Syntax
+
+```javascript
+cy.route2(url: RouteMatcher, response?: RouteHandler)
+
+// If you need to define a specific HTTP method
+cy.route2(method: string, url: RouteMatcher, response?: RouteHandler)
+```
+
+{% note info %}
+To learn more about types, check out [the repo](https://github.com/cypress-io/cypress/blob/0d60f7cd3ede4c5a79e151b646fa2377a7ddb16c/packages/net-stubbing/lib/external-types.ts#L209-L210) for more information.
+{% endnote %}
+
+## Arguments
+
+**{% fa fa-angle-right %} url**
+
+- **Description**: Defines a filter for incoming HTTP requests to decide which requests will be handled by this route.
+
+- **Expects**: `String` | `RegExp` | `Object` | [`StaticResponse`](#StaticResponse)
+
+- **Type**: [`RouteMatcher`](https://github.com/cypress-io/cypress/blob/0d60f7cd3ede4c5a79e151b646fa2377a7ddb16c/packages/net-stubbing/lib/external-types.ts#L118)
+
+- **Examples**:
+
+```js
+// String
+cy.route2('/users')
+
+// Regex
+cy.route2(/users\/\d+/)
+
+// Glob
+// Match all paths in a given pattern
+// Cypress uses minimatch to match glob patterns
+// https://github.com/isaacs/minimatch
+// Cypress uses minimatch to match glob patterns
+// https://github.com/isaacs/minimatch
+cy.route2('/users/**')
+
+// The following would match the previous glob:
+// https://localhost:8080/users/johnsmith
+// https://localhost:8080/users/profile/edit
+// https://localhost:8080/users/transaction?month=03&day=20
+
+// RouteMatcher
+// For matching on properties of the request besides the URL.
+cy.route2({
+  path: '/users/**',
+  method: 'POST',
+})
+
+// StaticResponse
+// Allows you to define a response
+// that will be sent back to the browser
+// to fulfill the request
+cy.route2('/users', {
+  body: '{ "message": "Custom message" }',
+  headers: {
+    accept: 'application/json'
+  },
+  statusCode: 200,
+  forceNetworkError: false
+})
```

Comment:
You're defining a response, so the `accept` header isn't used. Also, you can set `body` to an object to automatically set the content-type: json header.

```suggestion
cy.route2('/users', {
  body: { message: "Custom message" },
  statusCode: 200,
  forceNetworkError: false
})
```